manage.py

```diff
@@ -51,19 +46,14 @@ def test(all=False):
 @manager.command
 def scrape():
     """Start an infinte loop to scrape & categorize articles."""
-    iterations = 0
     categorizer = Categorizer()
 
     while True:
         logging.info("Starting scrape...")
-        if (iterations % ITERATIONS_BEFORE_RESET) == 0:
-            categorizer.reset_title_categories()
 
         scrape_articles(categorizer)
 
-        iterations += 1
-        logging.info("Finished scrape. Zzz...")
-        time.sleep(SCRAPE_INTERVAL)
+        logging.info("Finished scrape. Let's run it back...")
```

Comment:
Yeah. Isn't that what our scraper is supposed to do?

The new categorization runs slower, so sleeping will just add more time to an already slow process. Is there a reason we need a sleep at the moment since this is running on its own process?